UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs

```diff
@@ -223,10 +223,10 @@ public void Update()
 					return (damage);
 				}
 
-				lock (PresentTiles)
-				{
-					PresentTiles[damageableLayer].TryGetValue(cellPos, out TileLcation);
-				}
+				// lock (PresentTiles)
```

Comment:
What did this commented code use to do?

Reply:
nothing, so i just commented it out. Probably should just remove it, but i think it was bod that added it so need to ask him what it was going to be used for.

Reply:
@Bod9001

Reply:
yeah looks good to get rid of